interactors/addressNonceHandler.go

```diff
@@ -101,6 +102,14 @@ func (anh *addressNonceHandler) sendTransaction(ctx context.Context, tx *data.Tr
 	return anh.proxy.SendTransaction(ctx, tx)
 }
 
+func (anh *addressNonceHandler) isTxAlreadySent(tx *data.Transaction) bool {
+	oldTx, nonceAlreadyUsed := anh.transactions[tx.Nonce]
+
+	return nonceAlreadyUsed && bytes.Compare(oldTx.Data, tx.Data) == 0 &&
+		oldTx.SndAddr == tx.SndAddr &&
```

Comment:
forgotten the check to receiver, gas limit & gas price. Value should be compared against nil and then you should use big.Cmp function

Reply:
done. For the moment we will not check gas limit & price